src/main/java/org/molgenis/vcf/decisiontree/runner/DecisionTreeFactoryImpl.java

```diff
@@ -46,11 +52,32 @@
   public DecisionTree map(VcfMetadata vcfMetadata, Settings settings) {
     ConfigDecisionTree configDecisionTree = settings.getConfigDecisionTree();
     Map<String, Label> labelMap = mapLabels(configDecisionTree.getLabels());
-    Map<String, Node> nodeMap = mapNodes(vcfMetadata, configDecisionTree.getNodes(), labelMap);
+    Map<String, List<String>> filesMap = mapFiles(configDecisionTree.getFiles());
+    Map<String, Node> nodeMap = mapNodes(vcfMetadata, configDecisionTree.getNodes(), labelMap, filesMap);
     Node rootNode = nodeMap.get(configDecisionTree.getRootNode());
     return DecisionTree.builder().rootNode(rootNode).build();
   }
 
+  private Map<String, List<String>> mapFiles(Map<String, Path> files) {
+    Map<String, List<String>> filesMap;
+    if (files == null) {
+      filesMap = emptyMap();
+    } else {
+      filesMap = new HashMap<>();
+          files.entrySet().stream()
+              .forEach(entry -> filesMap.put(entry.getKey(), this.mapFile(entry.getValue())));
+    }
+    return filesMap;
+  }
+
+  private List<String> mapFile(Path value) {
+    try {
+      return Files.readAllLines(value);
```

Comment:
Should we exclude empty lines at the end of the file?
```
ABC
DEF

```

Reply:
I'd say empty lines in general